### PR TITLE
tools:fix fpgadiag file access mode

### PR DESCRIPTION
**Pull request by anandaravuri**

### CODE REVIEW

File: tools/libboard/board_common/board_common.c [hunk 1]
```diff
@@ -73,7 +73,7 @@ fpga_result read_sysfs(fpga_token token, char *sysfs_path,
 		resval = res;
 		goto out_destroy;
 	}
-	printf("size=%d \n", size);
+
```
Comment (tswhison) [inline tools/libboard/board_common/board_common.c:76]: Could you submit this one in a separate pull request?

Comment (anandaravuri) [reply tools/libboard/board_common/board_common.c]: Removed from Merge request
